### PR TITLE
Fail early - ensure minimal performance

### PR DESCRIPTION
**Pull request by Bischoff**
## What does this PR change?

This PR changes the default timeout for build validations from half an hour (!) to 8 minutes.

Rationale:
 * fail early
 * ensure minimal performance
 
**Fail early**: the less we wait for failures, the less we lose our time. Today I lost one hour and a half for 3 failed bootstraps...

**Ensure minimal performance**: with a default timeout of half an hour, any action that takes, say, 25 minutes, will be considered as acceptable. Our customers will certainly not be of that opinion.

If SUSE Manager can't cope with parallel bootstraps in a reasonable delay, then it is a product bug, that should be addressed as such. We should not hide that problem in the test suite.

The comment says:
```
  # Build Validations will require longer timeouts due to the low performance of our VMs
  # if we ever improve this fact, we can reduce these timeouts.
```
If that is really the cause for longer timeouts, then accepting VMs that are twice as slow is generous enough, I think.

## Links

Ports:
* 4.0:
* 4.1:

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by calancha — APPROVED** [review]

**Review by srbarrios — COMMENTED** [review]
I'm fine to merge it ONLY if we first do a full pass of the BV test suite in parallel and this change doesn't impact.
If it does, failing applying states on bootstrap in any client, I would not agree on merging it until we don't address the product issue causing this. 

The issue we tried to fix with a bigger timeout was reported [here]( https://github.com/SUSE/spacewalk/issues/12775#issuecomment-737962684) and more than one developer debug it, but we could not found the root cause. So we took that decision to unblock the test of testsuite.

If you really want to merge this timeout in BV without further considerations, then make this value parametrizable and set up the value from the Jenkins Pipeline, depending on the mode of running (sequential/parallel) and being able to setup it when we run a feature manually with cucumber comand.

File: testsuite/features/support/env.rb
Comment (srbarrios) [inline testsuite/features/support/env.rb:41]:
```suggestion
  DEFAULT_TIMEOUT = ENV['CUSTOM_TIMEOUT'].to_i if ENV['CUSTOM_TIMEOUT']
```

**Review by lkotek — APPROVED** [review]
Very reasonable :-)

**Review by srbarrios — CHANGES_REQUESTED** [review]
Please don't merge this without my suggestion.
Once done, I can do another PR to take care of Pipeline part.

**Pull request closed without merging**
